courses/serializers.py

```diff
@@ -3,11 +3,12 @@
 """
 import json
 
+from django.core.exceptions import ImproperlyConfigured
 from django.core.urlresolvers import reverse
 from rest_framework import serializers
 import six
 
-from .models import Course, Module
+from .models import Course, Module, EdxAuthor
```

Comment:
nit: since this might grow, it can be a good idea to change it to 
```
from .models import (
    Course, 
    Module, 
    EdxAuthor
)
```
but I leave this up to you